tutorials/screenrecorder.py

```diff
@@ -0,0 +1,129 @@
+"""
+    mslib.msui.screenrecorder
+    ~~~~~~~~~~~~~~~~~~~
+
+    This python script is meant for recording the screens while automated  tutorials are running.
+
+    This file is part of mss.
+
+    :copyright: Copyright 2021 Hrithik Kumar Verma
+    :copyright: Copyright 2021 by the mss team, see AUTHORS.
+    :license: APACHE-2.0, see LICENSE for details.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+"""
+import numpy as np
+import cv2
+import datetime
+import os
+import sys
+import time
+import mss
```

Comment:
I guess that this is another mss, with an identical name as our project, or?

Reply:
Yeah. This mss is (multi screen shot) Its used for grabbing images while recording. Since, it is a cross platform compatible, I used this. (And not pillow)